Sources/PT.PM.MySqlParseTreeUst/MySqlAntlrConverter.cs

```diff
@@ -0,0 +1,3034 @@
+                TextSpan = context.GetTextSpan()
+            };
+
+            var queryElements = new List<Expression>
+            {
+                context.uid().ConvertToInOutArgument()
+            };
+
+            int startIndex = context.children.IndexOf(context.uid()) + 1;
+
+            for (int i = startIndex; i < context.ChildCount; i++)
+            {
+                queryElements.Add((Expression)Visit(context.GetChild(i)));
+            }
+
+            result.QueryElements = queryElements;
+            return result;
+        }
+
+        public Ust VisitFileSizeLiteral([NotNull] MySqlParser.FileSizeLiteralContext context)
+        {
+            return VisitChildren(context);
+        }
+
+        public Ust VisitFlushOption([NotNull] MySqlParser.FlushOptionContext context)
+        {
+            return VisitChildren(context);
+        }
+
+        public Ust VisitFlushStatement([NotNull] MySqlParser.FlushStatementContext context)
+        {
+            return VisitChildren(context);
+        }
+
+        public Ust VisitFlushTableOption([NotNull] MySqlParser.FlushTableOptionContext context)
+        {
+            return VisitChildren(context);
+        }
+
+        public Ust VisitForeignKeyTableConstraint([NotNull] MySqlParser.ForeignKeyTableConstraintContext context)
+        {
+            return VisitChildren(context);
+        }
+
+        public Ust VisitFormatColumnConstraint([NotNull] MySqlParser.FormatColumnConstraintContext context)
+        {
+            return VisitChildren(context);
+        }
+
+        public Ust VisitFromClause([NotNull] MySqlParser.FromClauseContext context)
+        {
+            return VisitChildren(context);
+        }
+
+        public Ust VisitFullColumnName([NotNull] MySqlParser.FullColumnNameContext context)
+        {
+            return VisitChildren(context);
+        }
+
+        public Ust VisitFullColumnNameExpressionAtom([NotNull] MySqlParser.FullColumnNameExpressionAtomContext context)
+        {
+            return VisitChildren(context);
+        }
+
+        public Ust VisitFullDescribeStatement([NotNull] MySqlParser.FullDescribeStatementContext context)
+        {
+            return VisitChildren(context);
+        }
+
+        public Ust VisitFullId([NotNull] MySqlParser.FullIdContext context)
+        {
+            return VisitChildren(context);
+        }
+
+        public Ust VisitFunctionArg([NotNull] MySqlParser.FunctionArgContext context)
+        {
+            return VisitChildren(context);
+        }
+
+        public Ust VisitFunctionArgs([NotNull] MySqlParser.FunctionArgsContext context)
+        {
+            List<Expression> funcArgs = null;
+            var visitedArgs = (Expression)VisitChildren(context);
+            if (visitedArgs is MultichildExpression multichild)
+            {
+                var argList = ExtractMultiChild(multichild);
+                funcArgs = new List<Expression>(argList.Count);
```

Comment:
There is no need to convert `List` from `ExtractMultiChild` to `List`. Use only one instance.